sematic/db/schema.sql.sqlite

```diff
@@ -146,4 +149,6 @@ INSERT INTO "schema_migrations" (version) VALUES
   ('20221214142609'),
   ('20221215212459'),
   ('20230105163857'),
-  ('20230105170402');
+  ('20230105170402'),
+  ('20230217013045'),
+  ('20230217235835');
```

Comment:
Please add the other 2 migration scripts as well.

Reply:
Done.